apps/sms/views/shared/test/unit/system_message_handler_test.js

```diff
@@ -728,6 +730,24 @@ suite('SystemMessageHandler', function() {
       }).then(done, done);
     });
 
+    test('close the app when notification is removed with pending navigation',
+      function(done) {
+
+      Navigation.hasPendingInit.returns(true);
+      this.sinon.stub(window, 'close');
+
+      SystemMessageHandler.onNotificationSystemMessage({
+        // When notification is removed "clicked" property is "false".
+        clicked: false,
+        data: { id: smsMessage.id, threadId: smsMessage.threadId }
+      }).then(
+        () => new Error('Should not enter resolved case'),
+        () => {
```

Comment:
Not sure if it's meaningful to do so, because previously we didn't call toPanel while removing the notification either.